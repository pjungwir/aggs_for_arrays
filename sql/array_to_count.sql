SELECT array_to_count('{}'::float[]);
SELECT array_to_count('{1,1,5,2,0}'::smallint[]);
SELECT array_to_count('{1,1,5,2,0}'::integer[]);
SELECT array_to_count('{1,1,5,2,0}'::bigint[]);
SELECT array_to_count('{1,1,5,2,0}'::real[]);
SELECT array_to_count('{1,1,5,2,0}'::float[]);
SELECT array_to_count('{NULL}'::float[]);
SELECT array_to_count('{1,1,NULL,2,0}'::float[]);
SELECT array_to_count('{1,1,5,2,0}'::text[]);

