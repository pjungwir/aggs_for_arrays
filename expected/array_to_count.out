SELECT array_to_count('{}'::float[]);
 array_to_count 
----------------
               
(1 row)

SELECT array_to_count('{1,1,5,2,0}'::smallint[]);
 array_to_count 
----------------
              5
(1 row)

SELECT array_to_count('{1,1,5,2,0}'::integer[]);
 array_to_count 
----------------
              5
(1 row)

SELECT array_to_count('{1,1,5,2,0}'::bigint[]);
 array_to_count 
----------------
              5
(1 row)

SELECT array_to_count('{1,1,5,2,0}'::real[]);
 array_to_count 
----------------
              5
(1 row)

SELECT array_to_count('{1,1,5,2,0}'::float[]);
 array_to_count 
----------------
              5
(1 row)

SELECT array_to_count('{NULL}'::float[]);
 array_to_count 
----------------
              0
(1 row)

SELECT array_to_count('{1,1,NULL,2,0}'::float[]);
 array_to_count 
----------------
              4
(1 row)

SELECT array_to_count('{1,1,5,2,0}'::text[]);
 array_to_count 
----------------
              5
(1 row)

